pkgs/development/python-modules/blobfile/default.nix

```
@@ -0,0 +1,45 @@
+{ buildPythonPackage
+, fetchPypi
+, setuptools
+, urllib3
+, pycryptodomex
+, lxml
+, filelock
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "blobfile";
+  version = "2.0.1";
+
+  format = "wheel";
+
+  src = fetchPypi {
+    inherit pname version format;
+    python = "py3";
+    dist = "py3";
+
+    hash = "sha256-b1Gz6UjzCpLnNKl0sk/ND2pRhB/Qg96WiJkjFIE1jaI=";
+  };
```

```suggestion
  format = "setuptools";

  src = fetchFromGitHub {
    owner = "christopher-hesse";
    repo = "blobfile";
    rev = "v${version}";
    hash = "sha256-EUU/ZORqri3EIUUAho6cNdRgH+we13AgHyTjsY6tCWg=";
  };
```

It's a bit silly that they don't provide a source dist on pypi (only wheel), but building from GitHub is probably preferred to building from a wheel

(I don't feel that strongly about it, though, and an actual Python maintainer might have better input there)